### ISSUE TITLE
'brew tap' fails if osgeo/osgeo4mac not tapped

### ISSUE THREAD

**Issue by ArrEssJay** [qgis/homebrew-qgisdev#29]
Tapping the repository fails if osgeo/osgeo4mac is not already tapped.
I think the issue is this line:
`if build.with?("grass") || Formula["grass7"].opt_prefix.exist?`

This appears to be a race condition where the check on the formula is evaluated before the dependency checks leading to a syntax error before osgeo/osgeo4mac can be tapped.

If I remove `Formula["grass7"].opt_prefix.exist?` then  osgeo/osgeo4mac is tapped and other dependencies installed correctly.

```
Cthulu-Pro:~ rob$ brew -v tap qgis/qgisdev
==> Tapping qgis/qgisdev
git clone https://github.com/qgis/homebrew-qgisdev /usr/local/Homebrew/Library/Taps/qgis/homebrew-qgisdev --depth=1
Cloning into '/usr/local/Homebrew/Library/Taps/qgis/homebrew-qgisdev'...
remote: Counting objects: 22, done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 22 (delta 6), reused 7 (delta 3), pack-reused 0
Unpacking objects: 100% (22/22), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/qgis/homebrew-qgisdev/Formula/qgis3-dev.rb
No available formula with the name "grass7"
Error: Cannot tap qgis/qgisdev: invalid syntax in tap!
```

**Comment by dakcarto** [comment]
Hi @RobDeBagel, thanks for reporting!

Should be fixed now with 6a3ab49. Please try again. Thanks!

**Comment by ArrEssJay** [comment]
Confirmed, Thanks!
